packages/editor-ui/src/stores/cloudPlan.store.ts

```diff
@@ -180,6 +196,11 @@ export const useCloudPlanStore = defineStore(STORES.CLOUD_PLAN, () => {
 			console.warn('Error fetching user cloud account:', error);
 		}
 
+		const localStorageFlag = localStorage.getItem(LEAD_ENRICHMENT_FLAG);
```

Comment:
does there need to be also a PH flag for this feature?

Reply:
Checked with product, just adding and removing a survey question should be enough